platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init

```diff
@@ -16,9 +16,10 @@ start)
         echo -n "Setting up board... "
 
         depmod -a
-        modprobe i2c-dev
-        modprobe i2c-mux-pca954x
+	modprobe smc
 
+        (cd /usr/local/etc && ./lpc_cpld_x64_64 blu w 0x222 0x1f)
```

Comment:
cannot check-in a binary file.

Reply:
Our team need to use this tool to reset phy port. 
How can I use this tool ?